sources/packages/api/src/route-controllers/assessment/models/assessment.dto.ts

```diff
@@ -21,3 +23,16 @@ export class AssessmentHistorySummaryAPIOutDTO {
   studentAppealId?: number;
   studentScholasticStandingId?: number;
 }
+
+export class AssessmentNOAAPIOutDTO {
+  assessment: Assessment;
+  applicationNumber: string;
+  fullName: string;
+  programName: string;
+  locationName: string;
+  offeringIntensity: OfferingIntensity;
+  offeringStudyStartDate: string;
+  offeringStudyEndDate: string;
+  msfaaNumber: string;
+  disbursement: any;
```

Comment:
can we define a type for this or am i missing anything

Reply:
Not part of the PR but this is a dynamically built object that contains string and number properties. We can build a more typed object that would demand changes in the structure and on the NOA form definition.